includes/fs-plugin-info-dialog.php

```diff
@@ -955,18 +982,27 @@ function install_plugin_information() {
                 }
             }
 
-            // Add after $api->slug is ready.
-            $plugins_section_titles['features'] = fs_text_x_inline( 'Features & Pricing', 'Plugin installer section title', 'features-and-pricing', $api->slug );
+            if ( ! $is_theme ) {
+                // Add after $api->slug is ready.
+                $plugins_section_titles['features'] = fs_text_x_inline( 'Features & Pricing', 'Plugin installer section title', 'features-and-pricing', $api->slug );

```

Comment:
@fajardoleo any reason we include the features & pricing tab only for plugins and not for themes?

Reply:
Updated (the tab is relevant only to add-ons)